common/ayon_common/distribution/data_structures.py

```diff
@@ -12,55 +15,56 @@ class UrlType(Enum):
     SERVER = "server"
 
 
-@attr.s
-class MultiPlatformValue(object):
-    windows = attr.ib(default=None)
-    linux = attr.ib(default=None)
-    darwin = attr.ib(default=None)
+@dataclass
+class MultiPlatformValue:
+    windows: Union[str, None]
```

Comment:
Just out of curiosity, `Union[str, None]` is preferred/better from `Optional[str]`? 
It seems we are using both by looking at code.

Reply:
You have to provide default value if you use `Optional`.